compression/src/DemoAccount.sol

```diff
@@ -0,0 +1,98 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity >=0.7.0 <0.9.0;
+pragma abicoder v2;
+
+import {WaxLib as W} from "./WaxLib.sol";
+import {IDecompressor} from "./decompressors/IDecompressor.sol";
+
+contract DemoAccount {
```

Comment:
Demo code could be separate from other lib code, although this will occur once the lib contracts are separated.